### DISCUSSION TITLE
es/datasets/detect/coco/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#21997] [Docs]
# es/datasets/detect/coco/

Explora el conjunto de datos COCO para la detección y segmentación de objetos. Aprende sobre su estructura, uso, modelos preentrenados y características clave.

https://docs.ultralytics.com/es/datasets/detect/coco/

**Comment by williamrojas8928-commits** [comment]
If I want to train a custom model on COCO classes, such as person, car, and truck, do I need to train a new model from scratch? Or can I use a pre-trained model and just ignore the predictions of the other classes, even though this might involve more computational cost?

**Comment by williamrojas8928-commits** [comment]
If I want to train a custom model on COCO classes, such as person, car, and truck, do I need to train a new model from scratch? Or can I use a pre-trained model and just ignore the predictions of the other classes, even though this might involve more computational cost?

**Comment by UltralyticsAssistant** [comment]
👋 Hola, gracias por tu interés en Ultralytics 🚀. Este es un mensaje automatizado; un ingeniero de Ultralytics también te ayudará pronto. Recomendamos visitar la [Docs](https://docs.ultralytics.com/) para nuevos usuarios, donde encontrarás muchos ejemplos de [Python](https://docs.ultralytics.com/usage/python/) y [CLI](https://docs.ultralytics.com/usage/cli/) y donde puede que ya estén respondidas muchas preguntas comunes.

Si esto es un 🐛 Reporte de Bug, por favor comparte un [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) para ayudarnos a depurarlo.

Si es una ❓ Pregunta sobre entrenamiento personalizado, incluye la mayor información posible, como ejemplos de imágenes del dataset y logs de entrenamiento, y verifica que sigues nuestros [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Únete a la comunidad de Ultralytics donde te resulte más cómodo. Para chat en tiempo real, entra a [Discord](https://discord.com/invite/ultralytics) 🎧. ¿Prefieres debates más profundos? Visita [Discourse](https://community.ultralytics.com/). O participa en hilos de nuestro [Subreddit](https://www.reddit.com/r/Ultralytics/) para compartir conocimiento con la comunidad.

## Upgrade

Actualiza al último paquete `ultralytics`, incluyendo todos los [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) en un entorno [**Python>=3.8**](https://www.python.org/) con [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) para verificar que tu problema no esté ya resuelto en la versión más reciente:

```bash
pip install -U ultralytics
```

## Environments

YOLO puede ejecutarse en cualquiera de los siguientes entornos verificados y actualizados (con todas las dependencias incluyendo [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) y [PyTorch](https://pytorch.org/) preinstalados):

- **Notebooks** con GPU gratis: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. Ver [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. Ver [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. Ver [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

Si este badge está en verde, todas las pruebas de [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) están pasando. Las pruebas de CI verifican el funcionamiento correcto de todos los [Modes](https://docs.ultralytics.com/modes/) y [Tasks](https://docs.ultralytics.com/tasks/) de YOLO en macOS, Windows y Ubuntu cada 24 horas y en cada commit.
